workspace_host/interface.js

```diff
@@ -119,6 +117,24 @@ async
         workspace_server_settings.server_to_container_hostname =
           config.workspaceDevContainerHostname;
       }
+
+      // If we're not running in EC2, assume we're running with a local s3rver instance.
+      // See https://github.com/jamhall/s3rver for more details.
+      if (!config.runningInEc2) {
+        logger.verbose('Using local s3rver AWS configuration');
+        AWS.config.update({
+          s3ForcePathStyle: true,
+          accessKeyId: 'S3RVER',
+          secretAccessKey: 'S3RVER',
+          s3: {
+            endpoint: new AWS.Endpoint('http://localhost:5000'),
+          },
+        });
+      }
+
+      // It is important that we always do this:
+      // https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-region.html
+      AWS.config.update({ region: config.awsRegion });
```

Comment:
I didn't bring over the `AWS_ENDPOINT`/`awsServiceGlobalOptions` stuff as it was unused in the workspace host. _None_ of this will be here once we move to the v3 SDK, since there's no concept of global config.